helix-term/src/handlers/completion.rs

```diff
@@ -346,7 +368,14 @@ pub fn trigger_auto_completion(
                         ..
                     }) if triggers.iter().any(|trigger| text.ends_with(trigger)))
         });
-    if is_trigger_char {
+
+    let trigger_path_completion = matches!(
+        text.get_bytes_at(text.len_bytes())
+            .and_then(|t| t.reversed().next()),
+        Some(b'/' | b'\\')
```

Comment:
I think we only want to match the backslash on windows

Reply:
I don't think this should be platform specific. One could be writing code for mainly-Windows software (like PowerShell) while being on a Linux machine.

Reply:
> I think we only want to match the backslash on windows

Yeah I thought about doing this, but seemingly forgot about it not long after...

> One could be writing code for mainly-Windows software (like PowerShell) while being on a Linux machine

But on a linux machine there's no `\` as path separator is there? So it would not make sense to have this on Linux (as this path needs to exist and be readable for path-completion to work)?

Reply:
Treating `/` as `\` while serving completions on Linux is fine as Windows does the reverse, there's no need to complicate behavior for no benefit.

I'm sure someone out there is editing Windows specific source code on Linux, where they treat `foo/bar` as `foo\bar`

Reply:
Windows path syntax and Linux path syntax are not generally equivalent. The path detection is also OS dependent, backslashes act as escape chars (like with spaces) on unix but oath separators on windows.

Windows enables forward slashes as path separators in their paths so we support that but on unix you can absolutely not treat backslash as separators.

Some Windows paths also just don't make sense on unix. If you want crossplatform path completions always use forward slash since they work both on windows and unix (including powershell)

Reply:
Should be implemented now.

Reply:
Fair, I forgot that stuff like `cat foo\ bar` was valid posix shell for `cat "foo bar"`. I didn't know that `/` was that portable.